internal/pkg/tekton/pipeline_run_builder.go

```diff
@@ -156,6 +216,23 @@ func NewPipelineRunBuilder(name, namespace string) *PipelineRunBuilder {
 						},
 					},
 				},
+				Workspaces: []tektonv1.WorkspaceBinding{
+					{
+						Name: "shared-db",
+						VolumeClaimTemplate: &corev1.PersistentVolumeClaim{
```

Comment:
The data only need to be shared within this PipelineRun, an emptyDir volume is sufficient.